src/pgroonga-crash-safer.c

```diff
@@ -157,13 +157,15 @@ pgroonga_crash_safer_reset_position_one(Datum databaseInfoDatum)
 		int result;
 
 		SetCurrentStatementStartTimestamp();
-		result =
-			SPI_execute("SELECT proname "
-						"  FROM pg_catalog.pg_proc "
-						"  WHERE "
-						"    proname = 'pgroonga_wal_set_applied_position'",
-						true,
-						0);
+		result = SPI_execute(
+			"SELECT nspname AS schema_name "
+			"FROM pg_catalog.pg_namespace "
+			"WHERE oid in ("
+			"  SELECT pronamespace "
+			"  FROM pg_catalog.pg_proc "
+			"  WHERE proname = 'pgroonga_wal_set_applied_position')",
```

Comment:
SQLのフォーマットにすごいこだわりがあるわけじゃありませんが、既存のスタイルに揃えましょうか。

```suggestion
			"SELECT nspname "
			"  FROM pg_catalog.pg_namespace "
			"  WHERE oid in ("
			"    SELECT pronamespace "
			"    FROM pg_catalog.pg_proc "
			"    WHERE proname = 'pgroonga_wal_set_applied_position'"
			")",
```